templates/chrony.conf.epp

```diff
@@ -0,0 +1,122 @@
+# NTP servers
+<% $chrony::servers.each |$server| { -%>
+server <%= $server.flatten.join(' ') %>
```

Comment:
If `$chrony::servers` is an array, this is broken.
`Error: Error while evaluating a Method call, flatten(): Requires array to work with`

Also the old behavior was to use iburst
```
<% if @servers.is_a?(Hash) then @servers.keys.sort.each do |server| -%>
server <%= server %> <%= @servers[server].join(' ') %>
<% end else Array(@servers).each do |server| -%>
server <%= server %> iburst
<% end end -%>
```

Reply:
Actually... it might just be a problem because I was testing against a puppet 4 system... :(

Prior to puppet 5.5, `flatten` comes from stdlib.